server/src/test/java/io/druid/client/CachingClusteredClientTest.java

```diff
@@ -1430,6 +1438,184 @@ public void testTimeBoundaryCaching() throws Exception
     );
   }
 
+  @Test
+  public void testTimeSeriesWithFilter () {
```

Comment:
Please apply the standard IntelliJ formatting, I think it should get rid of that whitespace and maybe do other stuff too.